### ISSUE TITLE
Section > Responsive > Reverse not work for mobile, if I change the breakpoint.

### ISSUE THREAD

**Issue by healiny** [elementor/elementor#16322]
## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by healiny** [comment]
Hello
I changed the break point in mobile from 767 to 880.
https://www.screencast.com/t/grr9kgNkiIi

I Defined - Section> Responsive> Reverse Columns
https://www.screencast.com/t/I3AZ8d1N

Reverse Columns does not work in the 768 to 880 range. 
https://www.screencast.com/t/ccRYm3cZaJr
https://www.screencast.com/t/JmhzZ39hmH1

I found the bug for you 
https://www.screencast.com/t/vBOq6DivF

That's the code:
`@media (max-width: 767px)
    .elementor-reverse-mobile>.elementor-container>.elementor-row>:first-child {
    -webkit-box-ordinal-group: 11;
    -ms-flex-order: 10;
    order: 10;
}`

Regards
Eyal

**Comment by udidol** [comment]
Hi @healiny,
1. Did you save your site settings after changing the breakpoint values?
2. Did you try regenerating the CSS files after seeing that it doesn't work as expected?

To regenerate your CSS files, go to the Admin Dashboard > Elementor > Tools, and click on the "Regenerate files & data" button.

Once you get a green V sign, visit your front end page or edit the page with Elementor Editor, and check if the problem persists.
If the problem is solved, close the ticket. If it persists, please share your System Info Report.

**Comment by healiny** [comment]
Hi
1. The answer is yes.
2. I tried regenerating my CSS files as you wrote , and the problem is not solved.

I shared my System Info Report.
The page where the bug is located - 
https://dev.couples-harmony.com/%d7%90%d7%95%d7%93%d7%95%d7%aa/

**Comment by udidol** [comment]
@healiny
I see the problem. 
This is a bug related to the Optimized DOM experiment.
For now, until we resolve this issue, you can activate the Optimized DOM experiment to fix your problem.
To do that, you go to your Admin Dashboard > Elementor > Settings > Experiments > Optimized DOM > Activate > Save

**Comment by healiny** [comment]
@udidol 
It's indeed a bug and good that you saw too,
I Activateed the Optimized DOM and the issue is temporarily fixed.
The question is in which update will I know that the experiment is over, and I can disable the Optimized DOM?

**Comment by udidol** [comment]
@healiny
The experiments don't work that way.
Most if not all experimental features are intended to eventually be fully and permanently integrated into the product.
Until that happens, you will always be able to turn them on/off as you please.
You will get a notification on this issue once the fix for this bug is merged into the product. Once that happens, you can expect the fix to be in the coming release.

**Comment by healiny** [comment]
@udidol <https://github.com/udidol>
Thanks for your explanation.
I'm waiting for notification the fix for this bug.


‫בתאריך יום ד׳, 15 בספט׳ 2021 ב-9:39 מאת ‪udidol‬‏ <‪
***@***.***‬‏>:‬

**Comment by healiny** [comment]
@udidol 
You fixed the bug in Mobil and it works well.
There is another related problem, that the tablet has the exact same bug, needs to be fixed there as well. 
`https://www.screencast.com/t/MJPSMHFy`

**Comment by DorShahar** [comment]
This issue has been added to Elementor Core v3.5.0

Feel free to update

Thanks!
